health/health.go

```diff
@@ -734,17 +735,18 @@ func (t *Tracker) SetLatestVersion(v *tailcfg.ClientVersion) {
 	t.selfCheckLocked()
 }
 
-// SetCheckForUpdates sets whether the client wants to check for updates.
-func (t *Tracker) SetCheckForUpdates(v bool) {
+// SetAutoUpdatePrefs sets the client auto-update preferences.
+func (t *Tracker) SetAutoUpdatePrefs(check bool, apply opt.Bool) {
```

Comment:
not being able to use `AutoUpdatePrefs` due to the circular dependency is sad. We can fix that later, but meanwhile want to just document that the arguments correspond to `ipn.AutoUpdatePrefs`?